Pull in r198312 from upstream clang trunk (by Roman Divacky):

  Remove a tab that snuck in.

Introduced here: http://svnweb.freebsd.org/changeset/base/262262

Index: tools/clang/lib/Driver/Tools.cpp
===================================================================
--- tools/clang/lib/Driver/Tools.cpp
+++ tools/clang/lib/Driver/Tools.cpp
@@ -5786,7 +5786,7 @@ void freebsd::Assemble::ConstructJob(Compilation &
       CmdArgs.push_back("-matpcs");
     }
   } else if (getToolChain().getArch() == llvm::Triple::sparc ||
-	     getToolChain().getArch() == llvm::Triple::sparcv9) {
+             getToolChain().getArch() == llvm::Triple::sparcv9) {
     Arg *LastPICArg = Args.getLastArg(options::OPT_fPIC, options::OPT_fno_PIC,
                                       options::OPT_fpic, options::OPT_fno_pic,
                                       options::OPT_fPIE, options::OPT_fno_PIE,
